live-build/includes.chroot/usr/local/bin/compute-system-hash.sh

```diff
@@ -4,16 +4,34 @@ trap '' SIGINT SIGTSTP SIGTERM
 
 set -euo pipefail
 
-echo "TODO: Implement verify hash generation from vx-iso against installed image"
+#hardcoded to nvme for now
+mount -o ro /dev/nvme0n1p1 /mnt
 
-sleep 10
+# Check for the VxLinux-signed.efi and handle if it doesn't exist
+# That aside, we're using the strings command to extract the verity hash
+# from the signed efi binary to later use when calculating the SHV
+if [[ -f /mnt/EFI/debian/VxLinux-signed.efi ]]; then
+  VERITY_HASH=$(strings /mnt/EFI/debian/VxLinux-signed.efi | grep -o verity.hash=[a-zA-Z0-9]* | cut -d'=' -f2)
+else
+  VERITY_HASH=""
+fi
 
-exit 0
+umount /mnt
+
+if [[ ! -z "${VERITY_HASH}" ]]; then
+    if [[ `which xxd` && `which base64` ]]; then
+      base64_hash=$( echo -n ${VERITY_HASH} | xxd -r -p | base64 )
+      echo "Signed Hash: ${base64_hash}"
+      read -p "Press Enter once you have validated the Signed Hash."
```

Comment:
Let's refer to this hash as the system hash rather than the signed hash, to mirror the language that we use in vxsuite-complete-system and vxsuite.